tests/auth_tests/test_templates.py

```diff
@@ -52,3 +42,28 @@ def test_titles(self):
         response = PasswordChangeDoneView.as_view()(request)
         self.assertContains(response, '<title>Password change successful</title>')
         self.assertContains(response, '<h1>Password change successful</h1>')
+
+
+@override_settings(ROOT_URLCONF='auth_tests.urls')
+class AuthTemplateWithClientTests(TestCase):
+
+    def setUp(self):
+        self.pr_client = PasswordResetConfirmClient()
+        self.user = User.objects.create_user('jsmith', 'jsmith@example.com', 'pass')
+
+    def test_titles_with_bad_uidb64(self):
+        response = self.pr_client.get(reverse('password_reset_confirm',
```

Comment:
Use `url = reverse('password_reset_confirm', kwargs={'uidb64': uidb64, 'token': token}))` to avoid the funky indentation (we prefer hanging indent).